docs/en/frontend.md

```diff
@@ -0,0 +1,54 @@
+# Developer documentation
+
+## Example frontend app
+
+This module ships with an example frontend application written in React. This can be used as a guide/example of how
+to implement a decoupled frontend application for the CKAN dataset configuration set up in the backend.
```

Comment:
Along with this it can also be used out of the box and not look too bad. Maybe add a sentence being like "You don't have to implement your own application as this is used by default" or something